PyInstaller/loader/rthooks/pyi_rth_gstreamer.py

```diff
@@ -16,7 +16,7 @@
 # causes 100% CPU load. (Tested on OSX.)
 os.environ['GST_REGISTRY_FORK'] = 'no'
 
-os.environ['GST_PLUGIN_PATH'] = '{};{}'.format(
+os.environ['GST_PLUGIN_PATH'] = '{}:{}'.format(
     sys._MEIPASS, os.path.join(sys._MEIPASS, 'gst-plugins'))
```

Comment:
Ouch.
1) This should be `os.pathsep`, since that is it what it is for
2) Using format here is oversized. `os.pathsep.join()` would do the job, too.